core-localstorage.html

```diff
@@ -26,52 +26,95 @@
 
 <link rel="import" href="../polymer/polymer.html">
 
-<polymer-element name="core-localstorage" attributes="name value useRaw autoSaveDisabled" hidden>
 <script>
 
-  Polymer('core-localstorage', {
-    
+  Polymer({
+    is: 'core-localstorage',
     /**
      * Fired when a value is loaded from localStorage.
      * @event core-localstorage-load
      */
-     
-    /**
-     * The key to the data stored in localStorage.
-     *
-     * @attribute name
-     * @type string
-     * @default null
-     */
-    name: '',
-    
-    /**
-     * The data associated with the specified name.
-     *
-     * @attribute value
-     * @type object
-     * @default null
-     */
-    value: null,
-    
-    /**
-     * If true, the value is stored and retrieved without JSON processing.
-     *
-     * @attribute useRaw
-     * @type boolean
-     * @default false
-     */
-    useRaw: false,
-    
+
     /**
-     * If true, auto save is disabled.
-     *
-     * @attribute autoSaveDisabled
-     * @type boolean
-     * @default false
+     * Fired when localStorage was changed outside of current document (mirrors window storage event)
+     * @event core-localstorage-global-change
      */
-    autoSaveDisabled: false,
-    
+
+    published: {
+      /**
+       * The key to the data stored in localStorage.
+       *
+       * @attribute name
+       * @type string
+       * @default null
+       */
+      name: {
```

Comment:
I agree. That was the original API, I did not want to introduce any breaking changes.